doc/releases/changelog-dev.md

```diff
@@ -388,22 +393,40 @@ of operators. [(#2622)](https://github.com/PennyLaneAI/pennylane/pull/2622)
   ({-1: 470, 1: 530}, {-1: 470, 1: 530})
   ```
 
+* Quantum channels such as `qml.BitFlip` now support abstract tensors. This allows
```

Comment:
Needs its own category title within the new features section

Reply:
Yes I think we should group the last new features under a generic name 👍

Reply:
Hmm... having second thoughts here. What do you think? Is it actually part of operator arithmetic?

Reply:
Okay!

Reply:
Does this only support TF and Jax tensors? Were Torch tensors supported here previously?

Reply:
I.e. are the only "new" features here that we can now use quantum channels within QNodes that interface with TF or Jax?